app/controllers/v1/inventory_invitations_controller.rb

```diff
@@ -0,0 +1,41 @@
+class V1::InventoryInvitationsController < ApplicationController
+  before_action :authenticate_user!
+  authorize_actions_for :inventory
+
+  authority_actions create: 'update'
+  def create
+    create_params = invitation_params
+    create_params[:inventory_id] = inventory_id
+
+    send_invite = create_params.delete(:send_invite)
```

Comment:
This parameter surprised me, but I then realized that we were doing something similar [in the `ParticipantsController`](https://github.com/MobilityLabs/pdredesign-server/blob/master/app/controllers/v1/participants_controller.rb#L7).  What ultimately tipped me off was that we were passing in a boolean to send emails, although in practice, the check later only checks for a truthy value.

The likely purpose of this is to allow us to save without us automatically triggering emails, but if we don't provide that parameter, we introduce the state of us saying we've sent the email, but that hasn't actually happened.  This makes resending the email with the same parameters not currently possible.

Do we need this functionality right now?  I could see us adding it in later, but I'm not so sure we need it at this point in time.